app/grandchallenge/emails/tasks.py

```diff
@@ -76,20 +93,29 @@ def send_bulk_email(action, email_pk):
         messages = []
         for recipient in paginator.page(page_nr).object_list:
             user = get_user_model().objects.get(pk=recipient.pk)
+            link = reverse(

```

Comment:
FYI `action.html` is treated just like any other django template. You can load extensions, filters and do things in the template like normal, rather than doing it here.

Reply:
Yes, makes sense, but I think for the url it's just as easy to construct it here. Otherwise I would need to pass the username to the template.